wdl-lint/src/rules/malformed_lint_directive.rs

```diff
@@ -85,8 +85,9 @@ impl Rule for MalformedLintDirectiveRule {
     }
 
     fn explanation(&self) -> &'static str {
-        "Comments which begin with `#@` must only contain valid lint directives. Lint directives \
-         must be on their own line, only preceded by whitespace. Lint directives should follow the \
+        "This rule checks that lint directives (as defined in DEFINITIONS.md) are properly formatted. \
```

Comment:
This is going to be rendered in a variety of places (most notably `sprocket explain`) where it's not easy to determine where to look for `DEFINITIONS.md`. I'm not sure what an elegant solution is here... in the meantime I'd just remove this:
```suggestion
        "This rule checks that lint directives are properly formatted. \
```
We may want to print `DEFINITIONS.md` from `sprocket`, but I'm not sure the best way to accomplish that. I'll think on this issue some more.